pkg/parse/quote_test.go

```diff
@@ -47,7 +47,8 @@ func TestQuoteAs(t *testing.T) {
 		Args("a", SingleQuoted).Rets(`'a'`, SingleQuoted),
 		Args("\n", SingleQuoted).Rets(`"\n"`, DoubleQuoted),
 
-		// Bareword tested above in TestQuote.
+		// Bareword also tested above in TestQuote but verify invalid UTF-8 case here as well.
```

Comment:
These two things are unrelated, so don't write them in the same comment.

It's also better if you move this new test case above the original comment; these "foo tested elsewhere" comments should come last in a list of test cases.

```go
// Invalid UTF-8 sequences are quoted with \x
Args("bad\xffUTF-8", Bareword).Rets(`"bad\xffUTF-8"`, DoubleQuoted),

// Bareword tested above in TestQuote.
```

Reply:
Done